packages/codesniffer/Jetpack/ruleset.xml

```diff
@@ -44,6 +44,12 @@
 		<type>error</type>
 	</rule>
 
+	<!-- WordPress-Extra pulls this in. But PHP linting can be run more efficiently than having phpcs shell out to `php -l`. -->
+	<!-- Plus including it breaks code coverage generation with phpdbg. -->
+	<rule ref="Generic.PHP.Syntax">
+		<exclude name="Generic.PHP.Syntax" />
```

Comment:
I would vote against doing this. For other consumers of the coding package, this breaks the "WordPress CS + other things we think are good" approach. I think having the comment that this rule could be excluded if linted in another method, which is more performant.

I would vote *for* excluding this in the Jetpack plugin's root `.phpcs.xml.dist` since we have actively added a more efficient linting step.